recipes/sequana/meta.yaml

```diff
@@ -1,76 +1,88 @@
 {% set name = "sequana" %}
-{% set version = "0.7.1" %}
-{% set sha = "891e9a27fd992a9713de2e3afac9b4c8042d492703be65e1ae2b0ab098e275d6" %}
+{% set version = "0.8.2" %}
+{% set file_ext = "tar.gz" %}
```

Comment:
Please don’t introduce the `file_ext` and `hash_type` Jinja2 variables as they just complicate the recipe. Jinja2 variables should only be added if they are used at least twice.